contrib/intel/jenkins/common.py

```diff
@@ -23,37 +23,29 @@ def run_command(command):
         sys.exit(p.returncode)
 
 
-Prov = collections.namedtuple('Prov', 'core util')
+Prov = collections.namedtuple('Prov', 'core enable force_dl')
 prov_list = [
+    Prov("psm2", True, True), # psm2 is build with psm2 src as dl provider
+    Prov("psm3", True, False),
+    Prov("sockets", True, False),
+    Prov("verbs", True, False),
+#    Prov("gni", True, False),
+    Prov("udp", True, False),
+    Prov("tcp", True, False),
+    Prov("shm", True, False),
 
-   Prov("psm3", None),
-   Prov("psm2", None),
-   Prov("verbs", None),
-   Prov("verbs", "rxd"),
-   Prov("verbs", "rxm"),
-   Prov("sockets", None),
-   Prov("tcp", None),
-   Prov("udp", None),
-   Prov("udp", "rxd"),
-   Prov("shm", None),
-]
-enabled_prov_list = [
-    "psm2",
-    "verbs",
-    "tcp",
-    "sockets",
-    "udp",
-    "shm"
-]
-disabled_prov_list = [
-    "psm3",
-    'usnic',
-    'psm',
-    'efa',
-    'perf',
-    'rstream',
-    'hook_debug',
-    'bgq'
-    'mrail'
+    # Util Providers
+    Prov("rxm", True, False),
+    Prov("rxd", True, False),
+
+    # disabled Providers
+    Prov("psm", False, False),
+    Prov("efa", False, False),
+    Prov("usnic", False, False),
+    Prov("mrail", False, False),
+    Prov("bgq", False, False),
+    Prov("rstream", False, False),
+    Prov("perf", False, False),
+    Prov("hook_debug", False, False)
```

Comment:
separate from this PR, we need to understand why mrail, perf, and hook_debug are disabled.

Reply:
Ideally we should build with default configure options unless specified, I would think.